examples/react/react-app/src/App.tsx

```diff
@@ -23,25 +22,38 @@ function App() {
         isError: false,
     });
 
-    // get current component values
-    const position = dojo_starter.position.find({
-        player: account?.account.address,
-    });
-    const moves = dojo_starter.moves.find({ player: account?.account.address });
+    // Get current player entity
+    // This will return a player with the models that you specified
```

Comment:
I think it should return an array of the parts and then the dev can destructure the response like

```
let [ moves, position ] = ...
```

It is possible that models will have conflicting keys

Reply:
Yeah I was entertaining both ideas while creating the interface. Decided to go for the first solution because it seemed a bit cleaner if you had the full player directly at your disposal.

However I had indeed noticed that the typing would fall flat if the models had conflicting keys, where the return value would be of type `never`, which also isn't great DX.

Updated to the tuple approach!